src/app/admin/components/filter-search/filter-search.component.ts

```diff
@@ -72,13 +74,37 @@ export class FilterSearchComponent implements OnInit {
       s => !['rejected', 'unreleased'].includes(s.toLowerCase())
     );
     this.relevancyStart = new Date();
+
+    //check for params in the query and add them to the filter dropdown bars
+    const qParams= this.route.parent.snapshot.queryParams;
+    if(qParams.topics) {
+      if(Array.isArray(qParams.topics)) {
+        for(const topic of qParams.topics) {
+          this.toggleTopicFilter({ name: '', _id: topic });
+        }
+      } else {
+        this.toggleTopicFilter({ name: '', _id: qParams.topics });
+      }
+    }
+    if(qParams.status){
+      if(Array.isArray(qParams.status)) {
+        for(const status of qParams.status) {
+          this.toggleStatusFilter(status);
+        }
+      } else {
+        this.toggleStatusFilter(qParams.status);
+      }
+    }
+    if(qParams.collection){
+      this.toggleCollectionFilter(qParams.collection);
+    }
```

Comment:
Could you add a few more comments just describing the if statements above ^^

Reply:
gotcha